src/Makefile

```diff
@@ -1,5 +1,5 @@
 CC		= gcc
-CFLAGS		+= -O3 -Wall -DVERSION=\"0m/03\"
+CFLAGS		+= -O3 -Wall -DVERSION=\"1a/02\"
```

Comment:
Mentioned this in #16, which conflicts with this change. I'll leave discussion of the versioning approach to that branch!